ops/infrastructure/terraform.tf

```diff
@@ -132,7 +132,7 @@ module "vpc" {
   # You can enable communication from internet to RDS is via an internet gateway
   # to provide public access to RDS instance, but is not recommended for 
   # production! These parameters are all false so no public access to RDS
-  create_database_subnet_group = false
+  create_database_subnet_group = true
```

Comment:
Is it better to move `create_database_subnet_group = true` above the comment? As it has the value `true`, while comment mentions that all parameters are flase.

Reply:
This is done now.